object.js

```diff
@@ -0,0 +1,82 @@
+const projects = [
+  {
+    image: 'images/Snapshoot Portfolio.svg',
+    name: 'Tonic',
+    client: 'Canopy',
+    category: 'Back End Dev',
+    year: 2015,
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    technologies: ['HTML', 'CSS', 'JavaScript'],
+    liveLink: 'https://example.com/project-1-live',
+    sourceLink: 'https://github.com/user/project-1',
+  },
+  {
+    image: 'images/project2.jpg',
+    name: 'Tonic',
+    client: 'Client 2',
+    category: 'Mobile Development',
+    year: 2022,
+    description: 'Praesent vel massa quis mauris vehicula lacinia quis vel mi.',
+    technologies: ['React Native', 'Firebase'],
+    liveLink: 'https://example.com/project-1-live',
+    sourceLink: 'https://github.com/user/project-1',
+  },
+  {
+    image: 'images/project3.jpg',
+    name: 'Project 3',
+    client: 'Client 3',
+    category: 'Mobile Development',
+    year: 2015,
+    description: 'Praesent vel massa quis mauris vehicula lacinia quis vel mi.',
+    technologies: ['React Native', 'Firebase'],
+    liveLink: 'https://example.com/project-1-live',
+    sourceLink: 'https://github.com/user/project-1',
+  },
+  {
+    image: 'images/project4.jpg',
+    name: 'Project 4',
+    client: 'Client 4',
+    category: 'Mobile Development',
+    year: 2012,
+    description: 'Praesent vel massa quis mauris vehicula lacinia quis vel mi.',
+    technologies: ['React Native', 'Firebase'],
+    liveLink: 'https://example.com/project-1-live',
+    sourceLink: 'https://github.com/user/project-1',
+  },
+  // add more projects here...
+];
+
+const tonic = projects[0]; // Obtiene el proyecto "Tonic" del arreglo
+
+const tonicHTML = `
+  <article>
+    <img src="${tonic.image}" alt="${tonic.name}">
+    <div class="adjust">
+      <div class="leftblock">
+        <h2>${tonic.name}</h2>
+        <div class="ref">
+          <span class="client">${tonic.client}</span>
+          <span><i class="fa fa-circle fa-xs"></i></span>
+          <span>${tonic.category}</span>
+          <span><i class="fa fa-circle"></i></span>
+          <span class="year">${tonic.year}</span>
+        </div>
+        <p class="primarytext">${tonic.description}</p> 
+      </div>
+      <ul class="tags">
+        <li class="tag">${tonic.technologies[0]}</li>
+        <li class="tag">${tonic.technologies[1]}</li>
+        <li class="tag">${tonic.technologies[2]}</li>
+      </ul>
+      <div class="ref">
+        <button>See project</button>
+      </div>
+    </div>
+  </article>
+`;
+
+const projectsContainer = document.getElementById('projects-container');
+const tonicContainer = document.createElement('div');
+tonicContainer.innerHTML = tonicHTML;
+
+projectsContainer.appendChild(tonicContainer);
```

Comment:
- [ ] As stated in the project requirement Modal popup data Should be  populated with the data from the projects array.In the case of yours, your data is not Modal popup data is populated with the data from the projects array as shown below image.
- Your Popup window Look likes this : data is not populated from the array just like Figma design 
![Capture](https://user-images.githubusercontent.com/93598189/224509241-5bd9d165-61ad-4b00-8442-208e028c9666.PNG)

- The Figma Popup design Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/224509314-f8505441-c36d-4287-9076-6b9ab0c1c832.PNG)

- [ ] As shown in this [video](https://drive.google.com/file/d/1TWidr3nCCicS6OZOGZh_rwhMK-rWAV3Z/view?usp=share_link) the popups for both mobile and desktop screen sizes doesn't work as expected it only popup for one of the project list among the four list , So kindly Fix it.

Reply:
@Abdelaziz-Ali  Alright. I'll change it then. 

Thank you for your review.